LibreNMS/Data/Source/Fping.php

```diff
@@ -15,16 +15,17 @@
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
- * @link       https://www.librenms.org
- *
- * @copyright  2020 Tony Murray
+ * @package    LibreNMS
```

Comment:
You should fix your template :)
Remove package, it's some old deprecated stuff and update links to https

Reply:
Well, shoot, now I have to remember where that is :D